project-base/storefront/components/Blocks/Product/AddToCart.tsx

```diff
@@ -62,7 +62,6 @@ export const AddToCart: FC<AddToCartProps> = ({
             <Spinbox
                 defaultValue={1}
                 id={productUuid}
-                max={maxQuantity}
```

Comment:
I see several problems with the storefront code now, maybe we can take a look into in together or you can try @sebaholesz. The thing is, `<Spinbox>` component itself requires the `max` property to be set. I believe we want to keep that option so I would rather either change the places where product quantity is passed as maximum and change them for maximum integer, or (maybe even better) refactor `AddToCart` and `Spinbox` components so `max` is optional and applied only if set :thinking: